pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/model/CredentialsBindingHandler.java

```diff
@@ -69,6 +71,9 @@ public boolean handles(StandardCredentials c) {
     }
 
     @Nonnull
+    public abstract List<MultiBinding<C>> toBindings(String varName, String credentialsId);
```

Comment:
This can't be abstract as it is an addition to an already released API signature.